content/pt-br/docs/tutorials/kubernetes-basics/explore/explore-intro.html

```diff
@@ -16,124 +20,324 @@
+          </p>
+          <p><b><code>kubectl get pods</code></b></p>
+          <p>
+            Se nenhum Pod estiver rodando, aguarde alguns instantes e liste os
+            Pods novamente. Você pode continuar assim que ver um Pod rodando.
+          </p>
+          <p>
+            A seguir, para visualizar quais contêineres encontram-se no Pod e quais
+            imagens foram utilizadas para criar tais contêineres iremos usar o
+            comando <code>kubectl describe pods</code>:
+          </p>
+          <p><b><code>kubectl describe pods</code></b></p>
+          <p>
+            Vemos aqui detalhes a respeito dos contêineres do Pod: endereço IP,
+            portas utilizadas e uma lista de eventos relacionados ao ciclo de vida
+            do Pod.
+          </p>
+          <p>
+            A saída do subcomando <tt>describe</tt> é extensa e cobre alguns
+            conceitos que não foram explicados ainda. Não se preocupe, estes
+            conceitos serão familiares até o fim deste <i>bootcamp</i>.
+          </p>
+          <p><em>
+            <strong>Nota:</strong> o subcomando <tt>describe</tt> pode ser utilizado
+            para obter informações detalhadas sobre a maioria das primitivas do
+            Kubernetes, incluindo Nós (Nodes), Pods e Deployments. A saída do
+            subcomando <tt>describe</tt> é projetada para ser legível por humanos,
+            não para ser consumida por processos automatizados.
+          </em></p>
+        </div>
+      </div>
 
-            </div>
-            <div class="col-md-4">
-                <div class="content__box content__box_fill">
-                    <p><i>Um nó é uma máquina operária do Kubernetes e pode ser uma VM ou máquina física, dependendo do cluster. Vários Pods podem ser executados em um nó.</i></p>
-                </div>
-            </div>
+      <div class="row">
+        <div class="col-md-12">
+          <h3>Exiba a aplicação no terminal</h3>
+          <p>
+            Lembre-se que Pods estão executando em uma rede isolada e privada -
+            portanto, precisaremos de um proxy para acessá-los, a fim de tornar
+            possível a depuração e a interação com estes objetos. Para inicializar
+            um proxy, utilizaremos o comando <code>kubectl proxy</code> em um
+            <strong>segundo terminal</strong>. Abra uma nova janela do terminal,
+            e nesta nova janela, execute o comando:
+          </p>
+          <p><code><b>kubectl proxy</b></code></p>
+          <p>
+            Agora iremos novamente obter o nome do Pod e obter informações do Pod
+            diretamente através do proxy. Para obter o nome do Pod e armazená-lo
+            na variável de ambiente <tt>POD_NAME</tt>, utilize o comando:
+          </p>
+          <p>
+            <code>
+              <b>export POD_NAME="$(kubectl get pods -o go-template --template
+                '{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}')"</b>
```

Comment:
Hi, in the line 261 there is an white space between "range" and ".items", please, may you fix these?
Thanks

Reply:
Hi there, @jcjesus! Could you please point out as a suggestion here in GitHub which whitespace you're referring to?

If you mean `{{range .items}}` should be `{{range.items}}`, while that would still work, our Portuguese version would be diverging from the English version (and this command works with the whitespace present): https://github.com/kubernetes/website/blob/13a9d24f778372fece6b686300c6bbcbcdd8a6ed/content/en/docs/tutorials/kubernetes-basics/explore/explore-intro.html#L154